packages/ephemeral/indexer.js

```diff
@@ -5,11 +5,7 @@ module.exports = declareInjections({
   loadInitialModels: 'config:initial-models'
 }, class Indexer {
 
-  async branches() {
-    return ['master'];
-  }
```

Comment:
@robustdj this is an example of a change that should be made to any indexers. (we have a couple indexers in the various cards of portfolio). you'll need to delete the `branches()` hook that you find in any indexers.